### PR TITLE
Run panel PANELAPP-GREEN by default in all MIP runs

### PR DESCRIPTION
**Pull request by beatrizsavinhas**
## Description

Add panel PANELAPP-GREEN to all MIP runs.
Related to issue https://github.com/Clinical-Genomics/cg/issues/2698

### Added

- PANELAPP-GREEN to run by default

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [x] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b add-PANELAPP-GREEN-to-mip -a
    ```

### How to test

- [x] Attempt to run a case with a panel *not* in master list with cg and note the new panel being fetched from Scout, e.g.
	```Shell
	cg set case justhusky -g OPHTHALMO
	cg workflow mip-dna start justhusky -d
	```

### Expected test outcome

- [X] Check that the new panel is being fetched from Scout when running a case with a panel *not* in master list 
- [X] Take a screenshot and attach or copy/paste the output.

## Review

- [X] Tests executed by BSV
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [X] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by beatrizsavinhas** [comment]
@dnil Is this the correct way to do this? Could you help me with testing also? 🤔

File: cg/meta/workflow/mip.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def convert_panels(customer: str, default_panels: list[str]) -> list[str]:
                     all_panels.add(extra_panel)
 
         # add OMIM to every panel choice
-        all_panels.add(GenePanelMasterList.OMIM_AUTO)
+        all_panels.add(GenePanelMasterList.OMIM_AUTO.PANELAPP-GREEN)
```
Comment (dnil) [inline cg/meta/workflow/mip.py:185]: I'm not familiar with this syntax? I think what you may want is:
```suggestion
        all_panels.add(GenePanelMasterList.OMIM_AUTO)
        all_panels.add(GenePanelMasterList.PANELAPP-GREEN)
```

Comment (beatrizsavinhas) [reply cg/meta/workflow/mip.py:185]: Yes, that sounds better!

**Comment by beatrizsavinhas** [comment]
## Testing

```Shell
20d [hiseq.clinical@hasta:~] [S_base] 4m4s $ cg workflow mip-dna start justhusky -d
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Case justhusky exists in Status DB
Starting full MIP analysis workflow for case justhusky
Flow cell check is not applicable - the case is either down sampled or external.
Found file earlycasualcaiman/2019-05-08/HHT5NDSXX_HapMap_S26_L001_R1_001.fastq.gz
Found file earlycasualcaiman/2019-05-08/HHT5NDSXX_HapMap_S26_L001_R2_001.fastq.gz
Found file slowlycivilbuck/2019-05-08/HHT5NDSXX_HapMap_S27_L001_R1_001.fastq.gz
Found file slowlycivilbuck/2019-05-08/HHT5NDSXX_HapMap_S27_L001_R2_001.fastq.gz
Found file hugelymodelbat/2019-05-08/HHT5NDSXX_HapMap_S28_L001_R1_001.fastq.gz
Found file hugelymodelbat/2019-05-08/HHT5NDSXX_HapMap_S28_L001_R2_001.fastq.gz
Case justhusky exists in Status DB
Case justhusky exists in Status DB
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml export panel --bed BRAIN Cardiology CILM CH CTD DIAB ENDO EP HEARING HYDRO IBMFS IEM IF mcarta MHT MIT MOVE mtDNA NBS-M NEURODEG NMD OMIM-AUTO PANELAPP-GREEN PEDHEP PID PIDCAD RETINA SKD SOVM STROKE AID --build 37
```

Panel PANELAPP-GREEN loaded successfully!

**Review by diitaz93 — APPROVED** [review]
👍

**Pull request merged**